jcoin.datacommons.io/manifests/fence/fence-config-public.yaml

```diff
@@ -136,6 +136,9 @@ ITRUST_GLOBAL_LOGOUT: 'https://auth.nih.gov/siteminderagent/smlogout.asp?mode=ni
 LOGIN_OPTIONS:
   - name: 'Login from Google'
     idp: google
+  - name: 'ORCID Login'
```

Comment:
can we make the 2 names match in prod? either "login from x" or "x login" for both